src/Compilers/CSharp/Portable/Emitter/Model/SymbolAdapter.cs

```diff
@@ -18,8 +18,55 @@
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
-    internal partial class Symbol : Cci.IReference
+    internal abstract partial class
```

Comment:
I think for all of these types it would be worth a separate commit that reorders things so we don't switch between X and XAdapter constantly throughout the file. The structure you did here is good for the initial review as it makes it easier to understand what changed, but for future maintainability I think we should reorder the members (again, as a separate commit that consists just of moving things.

Reply:
> I think for all of these types it would be worth a separate commit that reorders things so we don't switch between X and XAdapter constantly throughout the file.

As I mentioned in another response, I am planning to consolidate conditional compilation in a different PR. 

---
In reply to: [512294623](https://github.com/dotnet/roslyn/pull/48838#discussion_r512294623) [](ancestors = 512294623)

Reply:
Ah, I missed that sentence. Thanks.

---
In reply to: [512683618](https://github.com/dotnet/roslyn/pull/48838#discussion_r512683618) [](ancestors = 512683618,512294623)